docs/hipify-clang.md

```diff
@@ -1,35 +1,35 @@
 # hipify-clang
 
-`hipify-clang` is a clang-based tool for translating CUDA sources into HIP sources.
-It translates CUDA source into an abstract syntax tree, which is traversed by transformation matchers.
-After applying all the matchers, the output HIP source is produced.
+`hipify-clang` is a clang-based tool for translating `CUDA` sources into `HIP` sources.
+It translates `CUDA` source into an abstract syntax tree (`AST`), which is traversed by transformation matchers.
+After applying all the matchers, the output `HIP` source is produced.
 
 **Advantages:**
 
-1. It is a translator; thus, any even very complicated constructs will be parsed successfully, or an error will be reported.
-2. It supports `clang` options like [`-I`](https://clang.llvm.org/docs/ClangCommandLineReference.html#cmdoption-clang-i-dir), [`-D`](https://clang.llvm.org/docs/ClangCommandLineReference.html#cmdoption-clang-d-macro), [`--cuda-path`](https://clang.llvm.org/docs/ClangCommandLineReference.html#cmdoption-clang-cuda-path), etc.
-3. Seamless support of new CUDA versions as it is the clang's responsibility.
-4. Ease of support.
+1. `hipify-clang` is a translator; any complex constructs will be parsed successfully, or an error will be reported.
+2. It supports `clang` options like [`-I`](https://clang.llvm.org/docs/ClangCommandLineReference.html#include-path-management), [`-D`](https://clang.llvm.org/docs/ClangCommandLineReference.html#preprocessor-options), [`--cuda-path`](https://clang.llvm.org/docs/ClangCommandLineReference.html#cmdoption-clang-cuda-path), etc.
+3. Seamless support of new `CUDA` versions, as the `clang` front end is statically linked into `hipify-clang` and does all the syntactical parsing of a `CUDA` source to hipify.
+4. Ease of support as a compiler extension.
 
 **Disadvantages:**
 
-1. The main advantage is also the main disadvantage: the input CUDA code should be correct; incorrect code wouldn't be translated to HIP.
-2. CUDA should be installed and provided in case of multiple installations by `--cuda-path` option.
+1. The main advantage is also the main disadvantage: the input `CUDA` code should be correct; incorrect code wouldn't be translated to `HIP`.

```

Comment:
@emankov What is the value add of this statement,

>The main advantage is also the main disadvantage:

How about,

Ensure the input `CUDA` code is correct; incorrect code will not be translated to `HIP`.